ios/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 2017-07-11 — LinkKit 1.0.6
+### Added
+- Add [copy customization](https://blog.plaid.com/link-copy-customization/) which allows to change the text of certain user interface elements in the Link flow
```

Comment:
nit: typo in `allows to change` – using copy from the blog why not say: `which gives you the ability to customize copy in the Link flow`.